src/screens/MainApp/MyDiagnoses/components/NoDiagnoses/index.js

```diff
@@ -1,36 +1,40 @@
 import React from 'react'
 import { SvgXml } from 'react-native-svg'
 import Background from '~/components/Background'
-import { Subtitle, Description } from '~/components/texts'
+import { Description } from '~/components/texts'
 import { PrimaryButton } from '~/components/buttons'
 import NavigationService from '~/navigationService'
-import icon from '~/assets/images/MyDiagnoses/NoDiagnosesIcon.svg'
+import outline from '~/assets/images/DetailedDiagnose/noCommentsYetOutline.svg'
 import styles, { ICON_WIDTH, ICON_HEIGHT } from './styles'
 
 const goToRequest = () => {
   NavigationService.navigate('DiagnoseRequest')
 }
 
-const NoDiagnoses = () => (
+const NoDiagnoses = ({ tab }) => (
```

Comment:
I already said this in another comment, but I think it'd be better to separate this component into two separate ones: `NoOpenDiagnoses` and `NoResolvedDiagnoses`.